spec/unit/presenters/v3/service_credential_binding_details_presenter_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+require 'presenters/v3/service_credential_binding_details_presenter'
+
+module VCAP
+  module CloudController
+    RSpec.describe Presenters::V3::ServiceCredentialBindingDetailsPresenter do
+      let(:instance) { ServiceInstance.make(guid: 'instance-guid') }
+      let(:app) { AppModel.make(guid: 'app-guid', space: instance.space) }
+      let(:json_creds) { { password: 'super secret avocado toast' }.to_json }
+      let(:credential_binding) do
+        ServiceBinding.make(
+          name: 'some-name',
```

Comment:
Doesn't the blueprint fill in `name` and `guid` if they are not set? If not, then we should fix the blueprint